### PR TITLE
Fix the net_edit script issue

### PR DESCRIPTION
**Pull request by yalzhang**
For negative test, do not need to check the network status after the
test. And as uuid is an attribute, the cmd should be different.

Signed-off-by: yalzhang <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
run all cases locally successfully

JOB ID     : 300659f95b4ef79714d49e4a0fde714052f18f29
JOB LOG    : /root/avocado/job-results/job-2018-12-05T15.00-300659f/job.log
 (1/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_define.modify.changeable: PASS (8.31 s)
 (2/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_define.modify.unchangeable: PASS (7.16 s)
 (3/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_define.modify.readonly_test: PASS (29.13 s)
 (4/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_define.delete.changeable: PASS (8.49 s)
 (5/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_define.delete.unchangeable: PASS (29.52 s)
 (6/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_create.modify.changeable: PASS (8.68 s)
 (7/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_create.modify.unchangeable: PASS (9.26 s)
 (8/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_create.delete.changeable: PASS (9.25 s)
 (9/9) type_specific.io-github-autotest-libvirt.virsh.net_edit.net_create.delete.unchangeable: PASS (28.24 s)
RESULTS    : PASS 9 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 143.60 s

File: libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py [hunk 1]
```diff
@@ -107,6 +108,11 @@ def edit_net_xml(edit_cmd, expect_error, **dargs):
     # Run test case
     try:
         libvirtd = utils_libvirtd.Libvirtd()
+        if attribute == "uuid":
+            # if the attribute need to change is uuid, the old uuid should get
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py]: if the attribute ...  to If  the uuid attribute need to be changed

File: libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py [hunk 2]
```diff
@@ -135,18 +141,19 @@ def edit_net_xml(edit_cmd, expect_error, **dargs):
...
         edit_net_xml(edit_cmd, status_error, readonly=readonly)
 
         net_state = virsh.net_state_dict()
         # transient Network become persistent after editing
-        # and the status should be the same as persistent network
-        if (not net_state[net_name]['active'] or
-                net_state[net_name]['autostart'] or
-                not net_state[net_name]['persistent']):
+        if not status_error and (not net_state[net_name]['active'] or
+                                 net_state[net_name]['autostart'] or
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py]: multiple or can be replaced by any() statement

File: libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py (hunk 1 above)
Comment (yalzhang) [reply libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py]: done and pushed, test locally and pass.

File: libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py (hunk 2 above)
Comment (yalzhang) [reply libvirt/tests/src/virsh_cmd/network/virsh_net_edit.py]: I tired, but I think the 2 method are similar, no one is simpler than the other.
if not A and (not B or C or not D):
and 
if not A and any(not B, C, not D)

**Pull request merged**